src/cartridge/include/cartridge/meta_cartridge.h

```diff
@@ -0,0 +1,86 @@
+/**
+ * @file meta_cartridge.h
+ * Declares the MetaCartridge class
+ */
+
+#include "cartridge/utils.h"
+
+#include <cstdint>
+#include <string>
+#include <vector>
+
+#pragma once
+
+namespace cartridge {
+
+class MetaCartridge {
+  public:
+	/**
+	 * The title of the Cartridge. Address from: 0x0134 to: 0x0143
+	 */
+	std::string title = "Welcome to TVP!";
```

Comment:
It seems strange to have the default Cartridge title be "Welcome to TVP", even if that's something we want to display when there is no title. It would be better to just give this no default, and in case there is no value for this, then set it from Video.

My reasoning for this is that no cartridge would actually have the game title named "TVP"